nevergrad/benchmark/plotting.py

```diff
@@ -248,12 +248,12 @@ def create_plots(
     os.makedirs(output_folder, exist_ok=True)
     # check which descriptors do vary
     descriptors = sorted(
-        set(df.columns) - (required | {"instrum_str", "seed", "pseudotime"})
+        set(df.columns) - (required | {"instrum_str", "seed", "pseudotime", "label"})
```

Comment:
what is `label`?
I expect this should not be here, but in the ExperimentFunction definition